### PR TITLE
Update GitHub Usage wording as per CELA recommendations

### PR DESCRIPTION
**Pull request by loic-sharma**
You can see the new `GitHub Usage` section here: https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784#evaluating-packages

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[822708a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/822708a52ccb999226ec8e2b8128629a4f785c35)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911182205161619-1784%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=NiTehyn0ZsI2WUKO%2FOXzMhXFFgFWP4MBNkIKCJI3%2F%2Bg%3D&st=2019-11-18T22%3A01%3A15Z&se=2019-12-19T22%3A06%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[8e6cd8c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/8e6cd8cd27ce51de5c06792170b67a06c41f6136)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911182208031042-1784%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6QpkzQAK%2Brh0HSeV3BcD0HKZaLHcPjck%2BBAsRD1gpmY%3D&st=2019-11-18T22%3A04%3A00Z&se=2019-12-19T22%3A09%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6919a28](https://github.com/NuGet/docs.microsoft.com-nuget/commits/6919a289c7a537ae8b7782d7d3e82024c7a580e4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911182210448397-1784%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qbofFXHQU%2FKEj42Ww2O7bPbkqaR9wh%2Fra6KEBBh5O3M%3D&st=2019-11-18T22%3A06%3A53Z&se=2019-12-19T22%3A11%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[159fd33](https://github.com/NuGet/docs.microsoft.com-nuget/commits/159fd33fd7b5f63e7da5ee9331ae5f14b48e9d77)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911182212502887-1784%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DlZNR2A%2FAvfmF09CA2xJb4CkFxsKuDKjjmP%2B1KsSys8%3D&st=2019-11-18T22%3A09%3A04Z&se=2019-12-19T22%3A14%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Finding-and-Choosing-Packages.md [hunk 1]
```diff
@@ -77,10 +77,13 @@ At the same time, using a NuGet package means taking a dependency on it, so you
 
     ![Download statistics on a package's listing page](media/Finding-03-Downloads.png)
 
-- *GitHub Usage*: on the package page, the **GitHub Usage** section lists the top GitHub repositories that depend on this package. A package that many popular GitHub repositories depend on is typically a better choice.
+- *GitHub Usage*: on the package page, the **GitHub Usage** section lists public GitHub repositories that depend on this package and that have a high number of stars on GitHub. A GitHub repository's number of stars generally indicates how popular that repository is with GitHub users (more stars usually means more popular). Please visit [here](https://help.github.com/en/github/getting-started-with-github/saving-repositories-with-stars#about-stars) for more information on GitHub's star and repository ranking system.
```
Comment (agr) [inline docs/consume-packages/Finding-and-Choosing-Packages.md]: Please, don't link the word "here".

Comment (agr) [reply docs/consume-packages/Finding-and-Choosing-Packages.md]: How about:
```
Please visit the GitHub's [getting started](https://help.github.com/en/github/getting-started-with-github/saving-repositories-with-stars#about-stars)...
```
?

File: docs/consume-packages/Finding-and-Choosing-Packages.md [hunk 2]
```diff
@@ -77,10 +77,13 @@ At the same time, using a NuGet package means taking a dependency on it, so you
 
     ![Download statistics on a package's listing page](media/Finding-03-Downloads.png)
 
-- *GitHub Usage*: on the package page, the **GitHub Usage** section lists the top GitHub repositories that depend on this package. A package that many popular GitHub repositories depend on is typically a better choice.
+- *GitHub Usage*: on the package page, the **GitHub Usage** section lists public GitHub repositories that depend on this package and that have a high number of stars on GitHub. A GitHub repository's number of stars generally indicates how popular that repository is with GitHub users (more stars usually means more popular). Please visit [GitHub's Getting Started page](https://help.github.com/en/github/getting-started-with-github/saving-repositories-with-stars#about-stars) for more information on GitHub's star and repository ranking system.
```
Comment (agr) [inline docs/consume-packages/Finding-and-Choosing-Packages.md]: I'd say, the density of using `GitHub` per sentence is a bit too high...

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[a1da92f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/a1da92f1d05d9e915a98800e85521837aad67adb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1784) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201911192156080294-1784%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1oX%2FryUln17ZQpWa4dIE7%2FVK6FLAAO%2F166gFnjiGvow%3D&st=2019-11-19T21%3A52%3A41Z&se=2019-12-20T21%3A57%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**